### DISCUSSION TITLE
Reaching auth endpoint

### DISCUSSION THREAD

**Discussion by 00Adnan00** [serverpod/serverpod#2515] [Q&A] [answered]
Serverpod provides a package on flutter side to handle lets say 'sign in with email' it provides a SignInWithEmailButton widget, But I want to implement my own button, any idea how I can reach AuthEndpoint?

I tried something like this:
<img width="747" alt="image" src="https://github.com/user-attachments/assets/29fa7af4-ec51-4a9d-b060-0537dbe0b382">
but AuthResponse in ambiguous

**Comment by SandPod** [comment, marked as answer]
Hello @00Adnan00!

These methods should be available under `client.modules.auth.*`. Did you add the `serverpod_auth_client` module as a dependency to your client?
